src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -84,7 +83,10 @@ for further details.
   height: 100%; /* for positioning the version switcher */
   overflow-y: auto; /* for positioning the version switcher */
   position: relative;
-  padding: 24px;
+  padding-bottom: 150px;
```

Comment:
💭 What do you think about changing this to `84px` so the padding all the way around the content is largely the same? It feels _off_ with significantly more bottom padding.

![Screen Shot 2022-06-03 at 3 31 36 PM](https://user-images.githubusercontent.com/1289701/171948032-37d2a6ac-a410-4448-bd3e-ff932d2cd7a5.png)
.

Reply:
I think it's better with more padding. It provides very clear separation from the version switcher and on other pages it keeps the content from being so close to the bottom of the window/screen.